charts/libsql/values.yaml

```
@@ -0,0 +1,79 @@
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 75
+  targetMemoryUtilizationPercentage: 75
+
+# Additional volumes on the output Deployment definition.
+volumes:
+ - name: data
+   emptyDir: {}
+
+# Additional volumeMounts on the output Deployment definition.
+volumeMounts:
+ - name: data
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)